analysisservice/src/test/java/nu/yona/server/analysis/service/AnalysisEngineServiceTest.java

```diff
@@ -104,6 +107,8 @@
 	@Mock
 	private ActivityCacheService mockAnalysisEngineCacheService;
 	@Mock
+	private GoalRepository mockGoalRepository;
```

Comment:
Same comment here, this is not a dependency of `AnalysisEngineService`...

Reply:
See reply above.